rust/parquet/src/arrow/schema.rs

```diff
@@ -184,10 +184,13 @@ fn get_arrow_schema_from_metadata(encoded_meta: &str) -> Option<Schema> {
             } else {
                 bytes.as_slice()
             };
-            let message = arrow::ipc::get_root_as_message(slice);
-            message
-                .header_as_schema()
-                .map(arrow::ipc::convert::fb_to_schema)
+            if let Ok(message) = arrow::ipc::root_as_message(slice) {
```

Comment:
We are ignoring errors here too. Is this the intent?

Reply:
DONE. Since there is already a eprintln!(), the new error is cached with eprintln!() too.
We may improve them later, right?

BTW, I failed to change the return type of `get_arrow_schema_from_metadata()` from `Option<Schema>` to `Result<Schema>` or `Result<Option<Schema>>`.